### ISSUE TITLE
problem with numbers

### ISSUE THREAD

**Issue by patej** [dilshod/xlsx2csv#64]
Hi,
I am facing a problem with xlsx files that have number such as 9.810070626E+019 (which shows in excel as 98100707530000000000) --> with xlsx2csv this becomes 98100707530000007168 when viewed with text editor. However, with OpenOffice xls-to-csv macro the value is shown as it should as 98100707530000000000.

Both of these are shown correctly when the csv is viewed in e.g. OpenOffice, but I'm taking the csv into different software, e.g. Pentaho Data Integration and there the values show as in a text editor..

Any help appreciated
--patrik

**Comment by Dongxn** [comment]
Did you solve this problem? I came to the same situation..

**Comment by patej** [comment]
No, I gave up and "solved" it by doing the file conversion directly with Pentaho.
